app/assets/javascripts/lat-long-info.js

```diff
@@ -0,0 +1,52 @@
+/* global FileReader EXIF $ */
+
+$(function () {
+  var sightingImageEl = document.getElementById('sighting_image')
+  if (sightingImageEl) {
+    sightingImageEl.addEventListener('change',
+    function (event) {
+      var fileInput = event.target
+      var imageFile = fileInput.files[0]
+
+      var fr = new FileReader()
+      fr.addEventListener('load', function () {
+        var thumbnailContainer = document.getElementById('img-thumb')
+        var thumbnail = document.createElement('img')
+        thumbnail.src = fr.result
+
+        thumbnailContainer.appendChild(thumbnail)
+
+        getExif(thumbnail, function (longitude, latitude) {
+          $('[name*=lat]').val(latitude)
+          $('[name*=long]').val(longitude)
+        })
+      })
+      fr.readAsDataURL(imageFile)
+    })
+  }
+})
+
+function getExif (img, cb) {
+  // if cb is not a function, log an error
+  if (typeof (cb) === 'function') {
+    EXIF.getData(img, function () {
+      var longitude = EXIF.getTag(this, 'GPSLongitude')
+      var latitude = EXIF.getTag(this, 'GPSLatitude')
+      // var latAndLong = document.getElementById('latAndLong')
+      // latAndLong.innerHTML = `${latitude},  ${longitude}`
```

Comment:
Don't leave deleted code as comments. Always just delete it. Make this an habit.